modin/pandas/base.py

```diff
@@ -3406,9 +3516,12 @@ def __init__(
         base=0,
         on=None,
         level=None,
+        origin: Union[str, TimestampConvertibleTypes] = "start_day",
+        offset: Optional[TimedeltaConvertibleTypes] = None,
     ):
         self._dataframe = dataframe
         self._query_compiler = dataframe._query_compiler
+        axis = self._dataframe._get_axis_number(axis)
```

Comment:
why not just `self._get_axis_number(axis)` ?

Reply:
Because `Resampler` has no `_get_axis_number()`.